src/api/useTerminologyServiceApi.ts

```diff
@@ -66,24 +68,29 @@ export class TerminologyServiceApi {
     }
   }
 
-  async getQdmValueSetsExpansion(cqmMeasure: CqmMeasure): Promise<ValueSet[]> {
+  async getQdmValueSetsExpansion(
+    cqmMeasure: CqmMeasure,
+    manifestExpansion: ManifestExpansion
+  ): Promise<ValueSet[]> {
```

Comment:
not related to your task and not sure why it did not complain but this promise of valuset looks incorrect as well. This ValueSet object should be imported from cqm-models.

Reply:
Good catch, its coming from R4, I thought this is custom model

Reply:
Changed it